python/tk_desktop/setup_project.py

```diff
@@ -46,11 +48,18 @@ def do_setup(self):
             if is_on_top: 
                 self._set_top_window_on_top(False)
 
-
             setup = adminui.SetupProjectWizard(self.project, self)
             ret = setup.exec_()
             self.setup_finished.emit(ret == setup.Accepted)
 
+        except TankErrorProjectIsSetup, e:
+            error_dialog = ErrorDialog("Toolkit Setup Error",
```

Comment:
:)
The plan is to bring up a dialog that will prompt for the force, but yup... in retrospect that would be a better way to go.